test/dotcom_web/views/partial_view_test.exs

```diff
@@ -130,9 +130,7 @@ defmodule DotcomWeb.PartialViewTest do
       }
 
       rendered =
-        news
-        |> news_entry()
-        |> safe_to_string()
+        news |> news_entry() |> Phoenix.HTML.Safe.to_iodata() |> IO.iodata_to_binary()
```

Comment:
Nit - it's slightly harder to read with all pipes on one line. Is this an automatic linter change?